index.html

```diff
@@ -67,5 +68,193 @@ <h1 class="social-media-heading">LET’S CONNECT</h1>
         </div>
       </section>
     </main>
+    <!-- The works pages -->
+    <section class="grid-container">
+      <section class="works">
+        <div class="tonic-1st">
+          <img class="tonic-1st-img" src="images/Snapshoot Portfolio.png" />
+          <h1 class="tonic-1st-heading">Tonic</h1>
```

Comment:
- [ ] Please it would be great if we can have any of the header tags `hx` aside `h1`. It's always professional to have just one `h1` header in a document for SEO optimization. ℹ️ 

> Note, kindly implement this for all the project title in your cards. 👍